boot/espressif/keys_sample/ec256-sample.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Any reason for this target to have it's own set of sample keys, instead of using the ones at the top of the tree?

Reply:
There isn't any special reason, I may did something wrong and when I tested it didn't work. I tested it again and works fine.
Should I change to use the keys in the root directory?

Reply:
I think it would be better to use the root directory keys. We have an issue #1094 about not storing these keys, but generating them when needed. It'll be easier to track down all the situations that do this if they are all at least using the same set of keys.

Reply:
Hi @d3zd3z, I've applied your suggestion and also removed those key samples. Please take a look.